src/lib/createTchapRoom.ts

```diff
@@ -23,41 +18,61 @@ export default class TchapCreateRoom {
         tchapRoomType: TchapRoomType,
         federate: boolean = DEFAULT_FEDERATE_VALUE): IOpts {
         const opts: IOpts = {};
-        const createRoomOpts: ITchapCreateRoomOpts = {};
+        const createRoomOpts: ICreateRoomOpts = {};
         opts.createOpts = createRoomOpts;
 
         //tchap common options
         createRoomOpts.name = name;
         opts.guestAccess = false; //guest access are not authorized in tchap
 
         createRoomOpts.creation_content = { 'm.federate': federate };
+        createRoomOpts.initial_state = createRoomOpts.initial_state || [];
 
         switch (tchapRoomType) {
             case TchapRoomType.Forum: {
             //"Forum" only for tchap members and not encrypted
-                createRoomOpts.accessRule = TchapRoomAccessRule.Restricted;
                 createRoomOpts.visibility = Visibility.Public;
                 createRoomOpts.preset = Preset.PublicChat;
+                createRoomOpts.initial_state.push({
```

Comment:
To set the access of a room, I have decided to use `createRoomOpts .initial_state` here instead of `createRoomOpts. accessRule` as we would need to :
- change the code in createRoom.ts and create tchapCreateRoom.ts (which should be ok)
- but we need also to change the code in `room-utils.ts` in particular :
```ts
return cy.createRoom(TchapCreateRoom.roomCreateOptions(roomName, TchapRoomType.External, false).createOpts);
```
This code is not taking into account the access rule settings and does not push the event to change the access rule therefore this part would need to change as well